docs/content/quatrix.md

```diff
@@ -0,0 +1,238 @@
+update it in rclone config. The same happens if the hostname was changed.
+
+```
+$ rclone config
+Current remotes:
+
+Name                 Type
+====                 ====
+remote               quatrix
+
+e) Edit existing remote
+n) New remote
+d) Delete remote
+r) Rename remote
+c) Copy remote
+s) Set configuration password
+q) Quit config
+e/n/d/r/c/s/q> e
+Choose a number from below, or type in an existing value
+ 1 > remote
+remote> remote
+--------------------
+[remote]
+type = quatrix
+host = some_host.quatrix.it
+api_key = jS2pbTDjp9VNlzq3PsSVgyxK2BgE423vcPV.N3CCmqyHiyCM
+--------------------
+Edit remote
+Option api_key.
+API key for accessing Quatrix account
+Enter a string value. Press Enter for the default (jS2pbTDjp9VNlzq3PsSVgyxK2BgE423vcPV.N3CCmqyHiyCM)
+api_key>
+Option host.
+Host name of Quatrix account
+Enter a string value. Press Enter for the default (some_host.quatrix.it).
+
+--------------------
+[remote]
+type = quatrix
+host = some_host.quatrix.it
+api_key = jS2pbTDjp9VNlzq3PsSVgyxK2BgE423vcPV.N3CCmqyHiyCM
+--------------------
+y) Yes this is OK
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+### Modified time and hashes
+
+Quatrix allows modification times to be set on objects accurate to 1 microsecond.
+These will be used to detect whether objects need syncing or not.
+
+Quatrix does not support SHA1 type hashes, so you cannot use the `--checksum` flag.
+
+### Restricted filename characters
+
+File names in Quatrix are case sensitive and have limitations like the maximum length of a filename is 255, and the minimum length is 1. A file name cannot be equal to `.` or `..` nor contain `/` or `\`.
+
+### Transfers
+
+For files above 50 MiB rclone will use a chunked transfer. Rclone will upload up to `--transfers` chunks at the same time (shared among all multipart uploads).
+Chunks are buffered in memory, and the minimal chunk size is 10_000_000 bytes by default, and it can be changed in the advanced configuration, so increasing `--transfers` will increase the memory use.
+The chunk size has a maximum size limit, which is set to 100_000_000 bytes by default and can be changed in the advanced configuration.
+The size of the uploaded chunk will dynamically change depending on the upload speed.
+The total memory use equals the number of transfers multiplied by the minimal chunk size.
+In case there's free memory allocated for the upload (which equals the difference of `maximal_summary_chunk_size` and `minimal_chunk_size` * `transfers`),
+the chunk size may increase in case of high upload speed. As well as it can decrease in case of upload speed problems.
+If no free memory is available, all chunks will equal `minimal_chunk_size`.
+
+### Deleting files
+
+Files you delete with rclone will end up in Trash and be stored there for 30 days.
+Quatrix also provides an API to permanently delete files and an API to empty the Trash so that you can remove files permanently from your account.
```

Comment:
Backends typically provide access to this with a `hard_delete` flag do delete stuff and not put it into the trash. Eg see `b2` backend. Rclone should default to `false` so make versions by default.

I don't think you need to implement this necessarily, but if it is easy (eg add a parameter to the DELETE call) then it would be a nice addition.

Reply:
I see you've done this :-)